sim/hunter/marksmanship/aimed_shot.go

```diff
@@ -0,0 +1,58 @@
+package marksmanship
+
+import (
+	"time"
+
+	"github.com/wowsims/cata/sim/core"
+	"github.com/wowsims/cata/sim/core/proto"
+)
+
+func (hunter *MarksmanshipHunter) registerAimedShotSpell(timer *core.Timer) {
+	if hunter.HasPrimeGlyph(proto.HunterPrimeGlyph_GlyphOfAimedShot) {
+		focusMetrics := hunter.NewFocusMetrics(core.ActionID{SpellID: 42897})
+		hunter.RegisterAura(core.Aura{
+		OnSpellHitDealt: func(aura *core.Aura, sim *core.Simulation, spell *core.Spell, result *core.SpellResult) {
+			if spell == hunter.AimedShot && result.DidCrit() {
+				hunter.AddFocus(sim, 5, focusMetrics)
+			}
+		},
+		})
```

Comment:
i might be remembering wrong but i think every aura needs atleast a `Label`

Reply:
Fixed